src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
+    config:set_integer("cluster", "n", NodeCount),
```

Comment:
Bob's last comment on that ticket sounds right to me. I'd have N set to:

```erl
N = case NodeCount of
    1 -> 1;
    NC when NC >= 3 -> 3;
end
```

Wherein bad cluster sizes (2, or < 1) are rejected (but I don't know what the project's stance on 2 node clusters is).